scripts/src/main/resources/scripts/command/terraform

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Check if terraform is installed.
+#
+function doIsTerraformInstalled() {
+  if command -v terraform &> /dev/null
+  then 
+    return
+  else
+    return 255
+  fi
+}
+
+# Call Terraform with specified arguments.
+function doRun() {
+  doEcho "Running: terraform ${*}"
+  terraform "${@}"
+}
+
+function doSetup() {
+
+  if doIsTerraformInstalled 
+  then
+    doEcho "terraform is already installed at $(command -v terraform)"
+    terraform version
+    echo
+  else
+      # Check if TERRAFORM_HOME is already set
+    #
+    if [ -z "$TERRAFORM_HOME" ]
+    then
+      doEcho "set TERRAFORM_HOME..."
+      export TERRAFORM_HOME="${DEVON_IDE_HOME}/software/terraform"
```

Comment:
Should we export it system-wide like I did for `GRAALVM_HOME`?

Reply:
I was not sure if there will somewhere started a sub shell. For that we need an export. I correct it